versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     okta = {
       source = "okta/okta"
-      version = "~> 3.17"
+      version = "3.30.0"
```

Comment:
Could you please add a comment here explaining why we need to pin to this version? You can include a link to the Okta provider GitHub issue as well. Thanks very much!

Reply:
Thanks for the suggestion @ccampo133, just added a comment contextualizing this change.